app/views/msb/branches_or_agents.scala.html

```diff
@@ -19,68 +19,41 @@
 @import forms2._
 @import models.autocomplete._
 
-@(f: Form2[_], edit: Boolean, countryData: Option[Seq[NameValuePair]])(implicit request: Request[_],m:Messages)
+@(f: Form2[_], edit: Boolean)(implicit request: Request[_],m:Messages)
 
 @header = {
     @components.back_link()
     @errorSummary(f)
-    @heading("msb.branchesoragents.title", "summary.msb")
 }
 
+@supportingContent = { <p>@Messages("msb.branchesoragents.intro")</p> }
```

Comment:
Has this been added to fieldsetWithErrorGroup?

Reply:
Whoops. I've added it now